src/main/java/mate/academy/model/User.java

```diff
@@ -13,7 +13,7 @@ public class User {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    @Column(unique = true)
+    @Column(length = 70, unique = true)
```

Comment:
```suggestion
    @Column(unique = true)
```
Why you changed it?

Reply:
![image](https://user-images.githubusercontent.com/99604928/191346147-eac97028-e683-4cfa-8cd7-43ff4a40b80c.png)
without limit

Reply:
Try @Column(unique = true, nullable = false)
If you still would have an exception leave 'length' parameter.

Reply:
didn't help :(